staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -225,6 +231,18 @@ type priorityLevelState struct {
 	// Periodically smoothed gets replaced with `max(envelope, A*smoothed + (1-A)*envelope)`,
 	// where A is seatDemandSmoothingCoefficient.
 	seatDemandStats seatDemandStats
+
```

Comment:
@MikeSpreitzer - can you please squash the last two commits - they are tightly coupled with each other (first doesn't make sense without the second).

Reply:
Done in the force-push to 722c85844e3